package-lock.json

```diff
@@ -6229,6 +6230,11 @@
         "url": "https://github.com/sponsors/d-fischer"
       }
     },
+    "node_modules/cropperjs": {
```

Comment:
cropper should be implemented the same way the other editors are implemented, without having to download anything additionally